biojava-genome/pom.xml

```diff
@@ -75,7 +75,7 @@
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
 			<scope>compile</scope>
-			<version>28.1-jre</version>
+			<version>30.1.1-jre</version>
```

Comment:
👍

Reply:
Wait, I am confused, it already is 30.1.1-jre on `master`.  See https://github.com/biojava/biojava/pull/928